azurerm/internal/services/network/firewall_resource.go

```diff
@@ -51,9 +51,39 @@ func resourceArmFirewall() *schema.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupName(),
 
+			// TODO 3.0: change this to required
```

Comment:
Does this default to a value right now? if so can we just defaut to it?

Reply:
Yes, the default sku name is: `AZFW_VNet`.

But persumably it will introduce a breaking change. Given a user has an existing FW with another sku name (because service is using another sku name at the time the FW is created), then the plan will mark the resource to be replaced.

WDYT?

Reply:
I think it would only affect users who have modified this resource outside of terraform or imported one that was created outside of terraform? but fair point - easy enough to mark it as computed till 3.0!